### PR TITLE
Relative Positioned Banner Animation (with Video) 

### PR DESCRIPTION
**Pull request by warkentien2**
here's the complete Banner Animation. 
I focus greatly on exactly mimicking the video guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Venus
```
Comment (arctouch-brunowolff) [inline README.md:1]: Nice ReadMe 👍

File: assets/bower.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (arctouch-brunowolff) [inline assets/bower.json]: Usually the bower.json file is located at root level, but it's not a rule...

File: index.html [hunk 3]
```diff
@@ -20,7 +20,35 @@
...
+        <img id="ice2" class="in-frame" src="assets/images/ice2.png" />
+      </div>
+      <div id="ice3-wrapper" class="in-frame no-overflow">
+        <img id="ice3" class="in-frame" src="assets/images/ice3.png" />
+      </div>
+      <div id="ice-top-wrapper" class="in-frame no-overflow">
+        <img id="ice-top" class="in-frame" src="assets/images/iceTop.png" />
+      </div>
+      <img id="logo" class="in-frame" src="assets/images/logo.png" />
+      <img id="razor1" class="in-frame razor" src="assets/images/razor.png" />
```
Comment (arctouch-brunowolff) [inline index.html]: Suggestion: I totally get the current scenario (and it's looking good ;] ), but using numbers (razor1, 2, 3) is a bit harder to get what's going on. Maybe renaming them to 'razorUpperCut' or 'firstShave' ( <- weir, I know).

File: assets/bower.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "assets",
```
Comment (arctouch-brunowolff) [inline assets/bower.json]: Usually the bower.json file is located at root level, but it's not a rule...

**Comment by arctouch-brunowolff** [comment]
Suggestion: I'd arrange the files a bit differently (placing logic/scripts within an 'asset' folder is not optimal): 

```
-- bower_components/
-- assets/
   |-- images/
-- styles/
-- scripts/

index.html
bower.json
readme.md
warkentien2-venus-banner.mkv 
```

**Comment by arctouch-brunowolff** [comment]
Reviewed 👍 Well done!

File: README.md (hunk 1 above)
Comment (warkentien2) [inline README.md:1]: thanks!

File: assets/bower.json (hunk 4 above)
Comment (warkentien2) [inline assets/bower.json]: I know, I was just trying to strictly follow the given instructions. (one main html + everything else in a folder)

File: index.html (hunk 3 above)
Comment (warkentien2) [inline index.html]: I agree, and will change this. I just lazily used the filenames.

**Comment by warkentien2** [comment]
I updated what was requested. I created a new pull request
https://github.com/warkentien2/venus/pull/3

**Comment by arctouch-brunowolff** [comment]
Superb!

**Pull request closed without merging**